### PR TITLE
Some more Agora updates

### PR DESCRIPTION
**Pull request by hewison-chris**

### CODE REVIEW

File: locales/en-US/en.yaml [hunk 1]
```diff
@@ -18,12 +18,12 @@ p2_text:
...
-  <a href="/register">Sign up</a> and add alerts on the 
-  <a href="/user/notifications">accounts page</a>. 
-  Get notified if your validators go offline. 
-  For more information about the beacon chain view our 
-  <a href="https://kb.beaconcha.in/">knowledge base.</a>'
+  <a href="/register">Sign up</a> and add alerts on the
+  <a href="/user/notifications">accounts page</a>.
+  Get notified if your validators go offline.
+  For more information about the beacon chain view our
+  <a href="https://kb.www.agorascan.io/">knowledge base.</a>'
```
Comment (TrustHenry) [inline locales/en-US/en.yaml:29]:
```suggestion
  <a href="https://www.agorascan.io/">knowledge base.</a>'
```

File: locales/ru-RU/ru.yaml [hunk 2]
```diff
@@ -14,16 +14,16 @@ p2_text:
...
-  <a href="/register">Вы можете зарегистрироваться</a> и включить уведомления для Вашего 
-  <a href="/user/notifications">аккаунта</a>. 
-  В случае если Ваши валидаторы отключатся, Вы будете получать уведомления. 
-  Для дополнительной информации о beacon chain зайдите в наш 
-  <a href="https://kb.beaconcha.in/">информационный центр.</a>'
+  <a href="/register">Вы можете зарегистрироваться</a> и включить уведомления для Вашего
+  <a href="/user/notifications">аккаунта</a>.
+  В случае если Ваши валидаторы отключатся, Вы будете получать уведомления.
+  Для дополнительной информации о beacon chain зайдите в наш
+  <a href="https://kb.www.agorascan.io/">информационный центр.</a>'
```
Comment (TrustHenry) [inline locales/ru-RU/ru.yaml:29]:
```suggestion
  <a href="https://www.agorascan.io/">информационный центр.</a>'
```

**Comment by TrustHenry** [comment]
There are many links that start like this.
`https://kb.`~

**Comment by hewison-chris** [comment]
> There are many links that start like this. `https://kb.`~

I think it is better to have broken link than to link to beaconcha.in pages. We can create the kb later.

File: templates/educationServices.html [hunk 3]
```diff
@@ -441,10 +441,10 @@ <h1 class="h2 mb-4 font-weight-bold text-primary text-center">Educational Materi
                   <td data-column="Support">N/A</td>
                 </tr>
                 <tr>
-                  <td data-column="Name"><a href="https://kb.beaconcha.in/">Beaconcha.in KB</a></td>
+                  <td data-column="Name"><a href="https://kb.www.agorascan.io/">www.agorascan.io KB</a></td>
                   <td data-column="Author"><a href="https://twitter.com/Butta_eth">Butta</a></td>
                   <td data-column="Type">General Knowledge</td>
-                  <td data-column="Association"><a href="https://twitter.com/beaconcha_in">beaconcha.in </a></td>
+                  <td data-column="Association"><a href="https://twitter.com/beaconcha_in">www.agorascan.io </a></td>
```
Comment (TrustHenry) [inline templates/educationServices.html:447]:
```suggestion
                  <td data-column="Association"><a href="https://t.me/bosagora_eng">www.agorascan.io </a></td>
```

File: locales/en-US/en.yaml (hunk 1 above)
Comment (hewison-chris) [reply locales/en-US/en.yaml:29]: I will update all `kb.www.agorascan.io` -> `kb.agorascan.io` so later we can create kb site with the updated Agora knowledge base pages.

Comment (TrustHenry) [reply locales/en-US/en.yaml:29]: 👍

**Comment by hewison-chris** [comment]
I added a new commit with the fixes.

File: templates/educationServices.html [hunk 4]
```diff
@@ -441,10 +441,10 @@ <h1 class="h2 mb-4 font-weight-bold text-primary text-center">Educational Materi
                   <td data-column="Support">N/A</td>
                 </tr>
                 <tr>
-                  <td data-column="Name"><a href="https://kb.beaconcha.in/">Beaconcha.in KB</a></td>
+                  <td data-column="Name"><a href="https://kb.agorascan.io/">www.agorascan.io KB</a></td>
                   <td data-column="Author"><a href="https://twitter.com/Butta_eth">Butta</a></td>
                   <td data-column="Type">General Knowledge</td>
-                  <td data-column="Association"><a href="https://twitter.com/beaconcha_in">beaconcha.in </a></td>
+                  <td data-column="Association"><a href="https://t.me/bosagora_eng">www.agorascan.io </a></td>
```
Comment (TrustHenry) [inline templates/educationServices.html:447]:
```suggestion
                   <td data-column="Association"><a href="https://twitter.com/bosagora1">www.agorascan.io </a></td>
```
I'm sorry. We had a Twitter site

**Review by TrustHenry — APPROVED** [review]
LGTM. Nice!

File: cmd/explorer/main.go [hunk 5]
```diff
@@ -230,7 +230,7 @@ func main() {
 
 		if !utils.Config.Frontend.OnlyAPI {
 			if utils.Config.Frontend.SiteDomain == "" {
-				utils.Config.Frontend.SiteDomain = "beaconcha.in"
+				utils.Config.Frontend.SiteDomain = "agora.in"
```
Comment (TrustHenry) [inline cmd/explorer/main.go:233]: `agora.in`
Don't you think this is a little weird?

**Pull request merged**